### PR TITLE
Fix 2 typos

### PR DESCRIPTION
**Pull request by nsoranzo**
Also update code for Python >=3.6 only

### CODE REVIEW

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: Python CI
 on: [push, pull_request]
+concurrency:
+  group: tox
```
Comment (mvdbeek) [inline .github/workflows/tox.yml:4]: Don't you need to include something that identifies a reference, so you don't cancel other PRs ?
We're using `group: <something>-${{ github.ref }}` in Galaxy for that reason.

File: gxformat2/cytoscape.py [hunk 2]
```diff
@@ -44,7 +42,7 @@ def to_cytoscape(workflow_path: str, output_path=None):
         tool_id = step.get("tool_id")
         if tool_id and tool_id.startswith(MAIN_TS_PREFIX):
             tool_id = tool_id[len(MAIN_TS_PREFIX):]
-        label = step.get("id") or step.get("label") or ("tool:%s" % tool_id) or str(i)
+        label = step.get("id") or step.get("label") or (f"tool:{tool_id}") or str(i)
```
Comment (mvdbeek) [inline gxformat2/cytoscape.py:45]:
```suggestion
        label = step.get("id") or step.get("label") or f"tool:{tool_id}"
```
The or `str(i)` was unreachable before, or am I missing something ?

File: scripts/new_version.py [hunk 3]
```diff
@@ -25,37 +25,33 @@ def main(argv):
     else:
         dev_version = re.compile(r'dev([\d]+)').search(version).group(1)
         new_dev_version = int(dev_version) + 1
-        new_version = version.replace("dev%s" % dev_version, "dev%s" % new_dev_version)
+        new_version = version.replace(f"dev{dev_version}", f"dev{new_dev_version}")
```
Comment (mvdbeek) [inline scripts/new_version.py:28]: This script has been copy-pasted between various projects, do we need to correct this in other places as well ?

File: gxformat2/cytoscape.py (hunk 2 above)
Comment (mvdbeek) [reply gxformat2/cytoscape.py:45]: Or maybe
```suggestion
        label = step.get("id") or step.get("label") or (f"tool:{tool_id}" if tool_id else str(i))
```
?

File: .github/workflows/tox.yml (hunk 1 above)
Comment (nsoranzo) [reply .github/workflows/tox.yml]: You're obviously right, I copy/pasted but deleted too much of the line. Fixed in a rebase.

File: scripts/new_version.py (hunk 3 above)
Comment (nsoranzo) [reply scripts/new_version.py:28]: It's not really a correction, I just modernised the code with pyupgrade now that we support Python >=3.6 only.
I think updating these scripts together with the rest of each project is OK, but we could surely update the "source" from where they are copied from, if there's one (@jmchilton should know).

Comment (jmchilton) [reply scripts/new_version.py:28]: Who knows 😆 - I just copied it from whatever project I had last touched probably. 

Thanks for the updates and fixes.

**Pull request merged**